test/integration/helpers/serverspec/platform_helper.rb

```diff
@@ -1,64 +1,69 @@
 # Encoding: utf-8
-require 'serverspec'
-require 'net/http'
-require 'openssl'
-
-require_relative 'php_examples'
-
 set :backend, :exec
 set :path, '/sbin:/usr/local/sbin:/bin:/usr/bin:$PATH'
 
-def redhat_family_values
-  res = {}
-  res['docroot'] = '/var/www/html'
-  res['apache_service_name'] = 'httpd'
-  res['fpm_service_name'] = 'php-fpm'
-  res['apache2ctl'] = '/usr/sbin/apachectl'
-  res
-end
-
-def debian_family_values
-  res = {}
-  res['docroot'] = '/var/www'
-  res['apache_service_name'] = 'apache2'
-  res['fpm_service_name'] = 'php5-fpm'
-  res['apache2ctl'] = '/usr/sbin/apache2ctl'
-  res
-end
-
-def ubuntu_trusty_family_values
-  res = {}
-  res['docroot'] = '/var/www/html'
-  res['apache_service_name'] = 'apache2'
-  res['fpm_service_name'] = 'php5-fpm'
-  res['apache2ctl'] = '/usr/sbin/apache2ctl'
-  res
-end
-
-def family_value(str)
-  if os[:family] == 'redhat'
-    redhat_family_values[str]
-  elsif os[:family] == 'ubuntu' && os[:release] == '14.04'
-    ubuntu_trusty_family_values[str]
-  else
-    debian_family_values[str]
-  end
-end
-
-def docroot
-  family_value('docroot')
-end
-
-def apache_service_name
-  family_value('apache_service_name')
-end
-
-def fpm_service_name
-  family_value('fpm_service_name')
+def test_data
```

Comment:
maybe I would call it differently but it seems clearer than before with this big hash!